app/assets/stylesheets/modules/panel_list.css.scss

```diff
@@ -1,4 +1,4 @@
-$panel-list-item-height: 85px;
+$panel-list-item-height: 75px;
```

Comment:
I see we are requesting *85px* images in ```collection.coffee```:
```
      if collection_id?
        "#{scalerUrl}/datasets/#{collection_id}?h=85&w=85"
      else if granule?
        "#{scalerUrl}/granules/#{granule}?h=85&w=85"
```
Should I update this to 75px? Or you are referring somewhere else?